csharp/csharp-core/intro-to-csharp/why-learn-csharp.md

```diff
@@ -11,20 +11,20 @@ category: must-know
 
 ---
 
-## Content
+# Content
```

Comment:
```suggestion
## Content
```

We only allow one H1 per insight, the content section must be H2 like before

Reply:
BTW You can read more about the format here: https://github.com/enkidevs/curriculum/wiki/Insight-Documentation

Reply:
Yay, I'll see and rewrite ig